app/src/main/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetUpView.kt

```diff
@@ -122,6 +126,27 @@ class MeetUpView : AppCompatActivity() {
         }
     }
 
+    /**
+     * Clicking on QR Code icon will show QR code
+     */
+    fun showQR(view: View?) {
```

Comment:
Why does this appear like new code when it was already implemented?

Reply:
because, as it is for meetup and we don't have the same identifier, the simplest way was to reimplement the showQr function of profileActivity